quantum/rgb_matrix_drivers.c

```diff
@@ -34,6 +34,7 @@ static void init(void) {
     IS31FL3731_init(DRIVER_ADDR_2);
 #    elif defined(IS31FL3733)
     IS31FL3733_init(DRIVER_ADDR_1, 0);
+    IS31FL3733_init(DRIVER_ADDR_2, 0);
```

Comment:
This should probably be more like: 
```suggestion
#        defined(DRIVER_ADDR_2) && DRIVER_ADDR_1 != DRIVER_ADDR_2
    IS31FL3733_init(DRIVER_ADDR_2, 0);
#        endif
```

Reply:
Would prefer to just change the offending boards instead of this hack.